.changeset/mighty-donkeys-sip.md

```diff
@@ -0,0 +1,17 @@
+---
+"@commercetools-backend/loggers": minor
+---
+
+Support to not unset the original field when rewriting logs.
+
+When rewriting a field you can use the `unsetFromField` option and set it to `false`. For example:
+
+```js
+{
+  from: 'meta.req.statusCode',
+  to: 'meta.req.status_code',
+  unsetFromField: false,
```

Comment:
```suggestion
Add support to optionally preserve the original field when rewriting log fields with `rewriteFieldsFormatter`.

When rewriting a field you can use the `preserveFromField` option and set it to `true`. For example:

```js
{
  from: 'meta.req.statusCode',
  to: 'meta.req.status_code',
  preserveFromField: true,
```